### PR TITLE
Refreshes user data on site refresh to fix role not updating issue-#22

### PR DESCRIPTION
**Pull request by SondreODahl**

### CODE REVIEW

**Comment by SondreODahl** [comment]
Closes #22

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -117,11 +129,21 @@ const App = () => {
...
             .then((res) => {
-                if(cognitoUserContainsAttributes(res)){
-                    dispatch(setUserInfo(res));
-                    dispatch(fetchOrganizationNameByID(res));
+                if (cognitoUserContainsAttributes(res)) {
+                    Auth.currentSession().then((currentSession) => {
+                        res.refreshSession(
+                            currentSession.getRefreshToken(),
+                            (err: any, session: any) => {
+                                console.log("session", err, session);
```
Comment (krisstam) [inline frontend/src/App.tsx:140]: Remove console.log :+1:

**Review by krisstam — COMMENTED** [review]
:+1:

**Review by krisstam — APPROVED** [review]
Looks good :+1:

**Pull request merged**